### ISSUE TITLE
Editing Code, Issues and everything is a HASSLE!

### ISSUE THREAD

**Issue by SecUpwN** [pockethub/PocketHub#570]
Dear @GitHub, as much as I love your platform, I begin to hate your App. This thing is just as inconvenient as can be, nothing feels comfortable and all editing has to be done through the tiny browser of my phone (which I have to force into desktop mode to even see any edit pen) while on the run. The current version of the GitHub App is just good for having a quick look into someones repository, but certainly not for any quality coding. If I open a file through your App, it is not editable at all.

I am sorry that my anger spills over into this Issue, but I would love to love GitHub on my Android as well - I simply can't! GitHub is the best platform to code with for me and I would like to dedicate more time to my Projects while in bed with my phone. Would you please simplify and restructure the GitHub App to conveniently display all features the web version of GitHub has? Don't be mad with me.

**Comment by ZarifinKolesi** [comment]
jonan/ForkHub nor pockethub/PocketHub has this issue.....

total ....... year 2017 ...........

**Comment by alfre344** [comment]
> I can understand editing text files, but I highly doubt you can do quality coding on a mobile device. Nevertheless I agree with you that there should be edit features in the app.

I agree. This function might come in handy in some cases where we need to make a simple change like an edit to some text typo to a release note or something similar.
